interpreter/cling/lib/Interpreter/IncrementalParser.cpp

```diff
@@ -390,6 +433,17 @@ namespace cling {
     return m_Consumer->getTransaction();
   }
 
+  void IncrementalParser::setDiagnosticConsumer(DiagnosticConsumer* Consumer,
+						bool Own) {
```

Comment:
Whitespace is a bit off here, maybe run through `clang-format` before merging?

Reply:
Fixed!  Thanks for spotting!